app/controllers/provider_interface/user_permissions_controller.rb

```diff
@@ -0,0 +1,57 @@
+module ProviderInterface
+  class UserPermissionsController < UsersController
+    def edit
+      @previous_page_path = previous_page_path
+      provider_permissions = @provider_user.provider_permissions.find_by!(provider: @provider)
+      @wizard = EditUserPermissionsWizard.from_model(edit_user_permissions_store, provider_permissions)
+    end
+
+    def update
+      @wizard = EditUserPermissionsWizard.new(edit_user_permissions_store, provider_permissions_params)
+      @wizard.save_state!
+
+      redirect_to check_provider_interface_organisation_settings_organisation_permissions_path(@provider, @provider_user)
+    end
+
+    def check
+      @wizard = EditUserPermissionsWizard.new(edit_user_permissions_store)
+    end
+
+    def commit
+      wizard = EditUserPermissionsWizard.new(edit_user_permissions_store)
```

Comment:
updating a record seems to be persisted, but isn't appearing on the summary page 🤔  happy to walk you through this

Reply:
That was a bug with the component displaying the permissions. Fixed in the latest commit